task/deprecated-image-check/0.1/deprecated-image-check.yaml

```diff
@@ -9,7 +9,8 @@ metadata:
   name: deprecated-image-check
 spec:
   description: >-
-    Checks whether base images are deprecated.
+    Identifies unmaintained and potentially insecure deprecated base images.
```

Comment:
Suggestion: Identify the unmaintained and potentially insecure deprecated base images with Pyxis API. The Pyxis API collects metadata from the image repository, and Conftest applies supplied policy on metadata.

Reply:
Even though I wanted to mention the tool used in the first sentence and not add a 2nd sentence at all but due to the complexity of the task I needed to separate it into 2 sentences.

So, I'd say we keep it as it is because the first sentence:

`Identify the unmaintained and potentially insecure deprecated base images with Pyxis API.`

implies that the identification is done using Pyxis API which is not true because Pyxis helps in supplying the metadata of image repositories and it is Conftest that uses our custom-written policies to point out the deprecated images using that metadata.